packages/lens/src/validate.ts

```diff
@@ -49,24 +50,63 @@ export async function getCollectAddress(postId: string) {
   return (collectAddress as Address) ?? null
 }
 
-export async function checkAddressOwnsCollect(
+export async function checkAddressMintedCollect(
+  actor: Address,
+  collectAddress: Address,
+) {
+  try {
+    return await checkMintedUsingBlockspan(actor, collectAddress)
+  } catch {
```

Comment:
how to deal with error handling here?

Reply:
If the second check fails we should throw `ValidationNotValid`
https://github.com/rabbitholegg/rh/blob/main/src/indexer/v4/consumers/sqs-plugin-action-validator.ts#L154C28-L154C46

Reply:
It should be caught [here](https://github.com/rabbitholegg/questdk-plugins/blob/060ee11f280706163b2925cee6bc90bd31ae608f/packages/lens/src/Lens.ts#L47-L48) and a `ValidationNotValid`. Is that okay or should I move this logic into the `validateCollect` function.